momepy/functional/_elements.py

```diff
@@ -152,6 +152,17 @@ def enclosed_tessellation(
         The number of jobs to run in parallel. -1 means using all available cores.
         By default -1
 
+    Warnings
+    --------
+    Due to the floating point precision issues in clipping the tessellation cells to the
```

Comment:
Should we include anything in here about the specific `shapely`/`GEOS` versions that lead to this issue?

Reply:
All of them from my testing.

Reply:
I tried some old gds_py containers with various versions of shapely and the problem was still there

Reply:
Same in the tessellation generated in 2020.

Reply:
😬 oh no